lib/tools/system-calls.js

```diff
@@ -199,7 +199,7 @@ systemCallMethods.getBinaryFromPath = async function getBinaryFromPath (binaryNa
 
 /**
  * @typedef {Object} ConnectedDevicesOptions
- * @property {?boolean} long [falsy] - Whether to get long output, which includes extra properties in each device.
+ * @property {?boolean} verbose [falsy] - Whether to get long output, which includes extra properties in each device.
```

Comment:
Default value notation might be skipped for these already containing a question mark